frontend/website/docs/getting-started.md

```diff
@@ -38,9 +38,9 @@ You can run `coda -help` to check if the installation succeeded.
 Add the Coda Debian repo and install.
 
 ```
-echo "deb [trusted=yes] http://packages.o1test.net unstable main" | sudo tee /etc/apt/sources.list.d/coda.list
+echo "deb [trusted=yes] http://packages.o1test.net release main" | sudo tee /etc/apt/sources.list.d/coda.list
```

Comment:
@yourbuddyconner or @jkrauska can one of y'all confirm that these commands are correct?

Reply:
`release` is the new repo for release builds